low-level/sensors/impl/centos_7/service_monitor.py

```diff
@@ -482,6 +485,10 @@ def raise_alert(self, service, prev_state, state, prev_substate,
                 }
             }
         }
+        if service == "kafka.service" and alert_type == "fault":
```

Comment:
add a separate function and raise IEM from there. This function should compare with a list of services for which we want to raise IEMs, even if its one service today

Reply:
Done.